docs/src/examples/components/Menu/Usages/MenuExampleToolbar.shorthand.tsx

```diff
@@ -66,26 +66,60 @@ const items = [
     'aria-label': 'Book Tool',
   },
   {
-    key: 'menuButton',
+    key: 'menuButton2',
     icon: {
       name: 'more',
       outline: true,
     },
     accessibility: toolbarButtonBehavior,
     'aria-label': 'More options',
     indicator: false,
-    menu: [
-      { key: '1', content: 'item1' },
-      {
-        key: '2',
-        content: 'item2',
-      },
-      {
-        key: '3',
-        content: 'item3',
-        menu: [{ key: '1', content: 'item3.1' }, { key: '2', content: 'item3.2' }],
-      },
-    ],
+    menu: {
+      items: [
+        {
+          key: '5',
+          content: 'item1',
+          icon: {
+            name: 'bookmark',
+            outline: true,
+          },
+        },
+        {
+          key: 'divider',
+          kind: 'divider',
+          isInSubMenu: true,
```

Comment:
This should be provided as a prop when the `MenuDivider` is created inside the `Menu` component, so that the users don't have to provide it..